ops/build.sh

```diff
@@ -1,13 +1,10 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
 
-# This script is used to compile the source code with AOT and
-# built it into an executable UberJar. For more information,
-# please check here:
-# - https://clojure.org/guides/deps_and_cli#aot_compilation
-# - https://github.com/tonsky/uberdeps
+# Build Uberjar with --main-class option. See for more
+# information, see https://github.com/tonsky/uberdeps
 
-# 1. Aot compile
-clj -e "(compile 'ptc.start)"
```

Comment:
Where does USE_CPCACHE come from?

Reply:
`USE_CPCACHE=0` ./ops/build.sh`

Reply:
In a case like this, where an env var is used effectively as a script argument, using an explicit script argument instead would make it clearer where this comes from, and how it's expected to be provided. At least, that what I thought when I saw this, not sure if this is was Tom is asking about.

Reply:
Or just pass CLJ_FLAGS through the environment.
Why test one variable only to set another?

Reply:
Because `USE_CPCACHE=0` is clearer at the call site than `CLJ_FLAGS=-Sforce`. Maybe in the future we want to pass more flags to `clojure` than just this one?

What I was trying to achieve was to pass a named argument like so:
`$ ./ops/build.sh USE_CPCACHE=0`

But I don't know how to implement that in bash and couldn't figure it out after trawling the internet for a couple of hours. If either of you can suggest an implementation, go for it.

Reply:
removing this flag - just going to build without using the cpcache

Reply:
Makes sense to me. "Though, given the problems we've seen, I'm tempted to just always run with the cache disabled."